examples/next-with-pages-router/src/pages/_app.tsx

```diff
@@ -0,0 +1,8 @@
+import '../styles/globals.scss';
+import type { AppProps } from 'next/app';
```

Comment:
is this correct order of imports? 🤔

Reply:
Looks like not. But it will be that we have the rule set up wrong in general for this case.